packages/nvidia_nat_core/tests/nat/authentication/test_oauth_exchanger.py

```
@@ -138,6 +140,20 @@ async def cb(conf, flow):
     assert calls["n"] == 1
 
 
+async def test_authenticate_uses_resolved_context_identity(monkeypatch, cfg):
+    """Token storage uses the authoritative runtime identity when no ID is passed."""
+
+    async def cb(conf, flow):
+        return _bearer_ctx(token="tok", expires_at=datetime.now(UTC) + timedelta(minutes=10))
```

Ruff (0.16.3)

[error] Possible hardcoded password assigned to argument: "token"

(S106)